src/spec/apis/system_api.py

```diff
@@ -51,3 +53,17 @@ async def ping_spec(
         return await self.ping(
             
         )
+
+    def toUuid(self, str: str) -> UUID:
+        """Translates str to UUID
+
+        Args:
+            str (str): str
+
+        Returns:
+            UUID: UUID
+        """
+        try:
```

Comment:
please insert missing tabs and remove extra space from str = str ( same as mentioned in earlier comment)